tests/test_circuits.py

```diff
@@ -166,3 +168,60 @@ def test_circuit_from_moments():
     moms = c.moments
     c2 = QCircuit.from_moments(moms)
     assert c == c2
+
+
```

Comment:
In case you have other 
standard gate = u123 with specific parameter choices identities it would be good to add them (the z test only tests for one specific choice, so for u3 the other two parameters could be implemented wrong and you woudn't notice since they are 0 anyway

maybe for
x,y,z,H
rx,ry,rz (with randomly chosen angles)

Reply:
I have added more tests for the u gates